tests/suites/spaces_ec2/task.sh

```diff
@@ -0,0 +1,27 @@
+test_spaces_ec2() {
+    if [[ "$(skip 'test_spaces_ec2')" ]]; then
+        echo "==> TEST SKIPPED: space tests (EC2)"
+        return
+    fi
+    if [[ "$BOOTSTRAP_PROVIDER" != "aws" ]]; then
```

Comment:
We need to update run.sh to sniff the cloud from `juju show-controller` when using an already bootstrapped controller...